VMEncryption/main/DiskUtil.py

```diff
@@ -397,44 +397,44 @@ def get_crypt_items(self):
 
                     crypt_items.append(crypt_item)
 
-            encryption_status = json.loads(self.get_encryption_status())
+        encryption_status = json.loads(self.get_encryption_status())
 
-            if encryption_status["os"] == "Encrypted" and not rootfs_crypt_item_found:
-                crypt_item = CryptItem()
-                crypt_item.mapper_name = "osencrypt"
+        if encryption_status["os"] == "Encrypted" and not rootfs_crypt_item_found:
```

Comment:
Can you write a detailed comment on the logic below? Difficult to understand just reading the code :)

Reply:
I'll try to parse and write it out.